common/server_player.cpp

```diff
@@ -1491,13 +1511,16 @@ Server_Player::cmdDelCounter(const Command_DelCounter &cmd, ResponseContainer &
 Response::ResponseCode
 Server_Player::cmdNextTurn(const Command_NextTurn & /*cmd*/, ResponseContainer & /*rc*/, GameEventStorage & /*ges*/)
 {
-    if (spectator)
-        return Response::RespFunctionNotAllowed;
-
     if (!game->getGameStarted())
         return Response::RespGameNotStarted;
-    if (conceded)
-        return Response::RespContextError;
+
+    if (!judge) {
+        if (spectator)
+            return Response::RespFunctionNotAllowed;
+
+        if (conceded)
```

Comment:
Is it intentional that a judge can change phase/turn of the game of a player after he conceded?
Afaik all the player zones and their contents should have been zapped already (Note: this comment applies to the following, similar changes, too).
This would also simplify the change to a simple
```
if(spectator && !judge)
..
```
If it's intentional, can you elaborate on a use case? Thank you

Reply:
In this flow the judge is executing the command directly, not though another player, so they will never be conceded.